resources/js/common/components/DiffTimestamp/DiffTimestamp.tsx

```diff
@@ -36,13 +36,19 @@ export const DiffTimestamp: FC<DiffTimestampProps> = ({
   }
 
   if (!enableTooltip) {
-    return <span className={className}>{diffForHumans(at, renderedAt)}</span>;
+    return (
+      <span className={className} suppressHydrationWarning>
```

Comment:
The linked document says to use `suppressHydrationWarning={true}`. Is the `{true}` implied?

Reply:
Yep -- React automagically passes boolean props without a given value as `true`.

Looking at this again though, I think I'd rather have it explicit than leave it to assumption. Updated in latest.